data/autoyast_sle15/autoyast_sle_powervm.xml.ep

```diff
@@ -88,13 +88,17 @@
     % foreach (values %$addons) {
         <package><%= lc($_->{name}) %>-release</package>
     % }
+      <package>firewalld</package>
     </packages>
     <patterns config:type="list">
       <pattern>apparmor</pattern>
       <pattern>base</pattern>
       <pattern>basesystem</pattern>
       <pattern>enhanced_base</pattern>
       <pattern>minimal_base</pattern>
+      <pattern>x11</pattern>
```

Comment:
I can see here more patterns than in https://openqa.suse.de/tests/11023045#step/disable_grub_timeout/1
we don't need to use .ep template to optimize from the beginning, the most important is that will be equivalent image created.

Reply:
Do you mean no need to add x11 here? The x11 pattern is one of the pattern from X Window System.